plugin/helpers_bots.go

```diff
@@ -18,11 +18,24 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 	}
 
 	// If we fail for any reason, this could be a race between creation of bot and
-	// retreval from anouther EnsureBot. Just try the basic retrieve existing again.
+	// retreval from another EnsureBot. Just try the basic retrieve existing again.
```

Comment:
Thanks for fixing the typo! While you're in here, can we also fix:
```suggestion
	// retrieval from another EnsureBot. Just try the basic retrieve existing again.
```